client/src/frontend/components/atomic/EntityHeadContent/index.js

```diff
@@ -25,10 +30,14 @@ function EntityHeadContent({ entity, parentEntity = { attributes: {} } }) {
 
   const title = () => {
     if (socialTitle) return socialTitle;
-    const title = parentTitle
-      ? `${parentTitle}: ${titlePlaintext}`
-      : titlePlaintext;
-    return `\u201c${title}\u201d on ${installationName}`;
+    const titleOrNum = (() => {
+      if (parentTitle) {
+        if (number) return `${parentTitle}: ${type} ${number}`;
+        return `${parentTitle}: ${titlePlaintext}`;
+      }
+      return titlePlaintext;
+    })();
+    return `\u201c${titleOrNum}\u201d on ${installationName}`;
```

Comment:
Since volumes and issues have titles not numbers, I've ended up doing a version of this in several places. Is there a better solution?

Reply:
I'm confused by this comment. Volumes and issues have numbers, not titles, right?

Reply:
Oh oops. Yep! I mean what you said. To elaborate, I am wondering if checking for `number` in these entity components is the right way to determine whether to render number or title for a given entity.